src/js/tracks/text-track-display.js

```diff
@@ -319,6 +320,45 @@ class TextTrackDisplay extends Component {
       }
       this.updateForTrack(descriptionsTrack);
     }
+
+    if (!window.CSS.supports('inset', '10px')) {
```

Comment:
since the current place where inset is used is `updateDisplayOverlay`, should this block go in there? There's already a check there for inset support.

Reply:
This condition is here because **updateDisplay** runs for every time the subtitles change unlike  **updateDisplayOverlay** & that is required since the subtitles change position a little bit every time they are generated.

Reply:
ah, i see. I guess the comment for it is incorrect, probably a copypasta from this method.